barretenberg/cpp/pil/vm2/bytecode/update_check.pil

```diff
@@ -8,7 +8,7 @@ include "../trees/public_data_check.pil";
 // Contract updates are written to the public data tree using the `shared mutable` structure.
 // The shared mutable structure looks like this:
 // +-------+---------------------+-----------------------+-----------------------+-------------------------+
-// | slot  | shared_mutable_slot | shared_mutable_slot+1 | shared_mutable_slot+2 |  shared_mutable_slot+3  |
+// | slot  | delayed_public_mutable_slot | delayed_public_mutable_slot+1 | delayed_public_mutable_slot+2 |  delayed_public_mutable_slot+3  |
```

Comment:
Nit: breaks the graph :) 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/mO08Q17TCoHzVKYudUJh/6453b6b5-b7c9-4d89-a15d-1ee60fc82b9f.png)